blocks/api/parser.js

```diff
@@ -220,7 +220,8 @@ export function createBlockWithFallback( name, rawContent, attributes ) {
  */
 export function parseWithGrammar( content ) {
 	return grammarParse( content ).reduce( ( memo, blockNode ) => {
-		const { blockName, rawContent, attrs } = blockNode;
+		// eslint-disable-next-line no-unused-vars
+		const { blockName, innerBlocks, innerHtml: rawContent, attrs } = blockNode;
```

Comment:
Should we replace all references to `rawContent` as `innerHTML` ? Here or in a separate pull changeset?

Reply:
or now I thought a second PR would be good so that this can be introduced in as minimal of a way as possible